src/Microsoft.AspNetCore.JsonPatch/Converters/TypedJsonPatchDocumentConverter.cs

```diff
@@ -60,5 +60,16 @@ public class TypedJsonPatchDocumentConverter : JsonPatchDocumentConverter
                 throw new JsonSerializationException(Resources.InvalidJsonPatchDocument, ex);
             }
         }
+
+        /// <summary>
+        /// Create the JsonPatchDocument using the appropriate constructor
+        /// </summary>
+        /// <param name="objectType">The model type for the JsonPatchDocument</param>
+        /// <param name="operations">The operations to assign to the JsonPatchDocument</param>
+        /// <returns>A new instance of the JsonPatchDocument</returns>
+        protected virtual object CreateTypedContainer(Type objectType, object operations)
+        {
+            return Activator.CreateInstance(objectType, operations, new DefaultContractResolver());
+        }
```

Comment:
Again, not totally sure what this is going to be used for... it would be good to talk through the scenarios.

Reply:
Same as above.  Just trying to allow custom Converters to be created with as little code duplication as possible.